scripts/run_mcmc.py

```diff
@@ -223,21 +223,21 @@ def run_chain(
             )
 
     # Make Move Probabilities
-    prune_and_reattach = config["move_probs"]["prune_and_reattach"]
-    swap_node_labels = config["move_probs"]["swap_node_labels"]
-    swap_subtrees = config["move_probs"]["swap_subtrees"]
+    prune_and_reattach = config.move_probs.prune_and_reattach
+    swap_node_labels = config.move_probs.swap_node_labels
+    swap_subtrees = config.move_probs.swap_subtrees
     move_probs = MoveProbabilities(prune_and_reattach, swap_node_labels, swap_subtrees)
 
     # run mcmc sampler
     mcmc_sampler(
         rng_key=rng,
         data=mut_mat,
-        error_rates=(config["fpr"], config["fnr"]),
+        error_rates=(config.fpr, config.fnr),
         move_probs=move_probs,
-        num_samples=config["num_samples"],
-        num_burn_in=config["burn_in"],
+        num_samples=config.n_samples,
+        num_burn_in=config.burn_in,
         output_dir=Path(params.out_dir),
-        thinning=config["thinning"],
+        thinning=config.thinning,
```

Comment:
Minor refactoring not planned for this PR - yet pyright did only catch it here.

Reply:
I see! I think it's a good change – stronger types are usually help with code readability and spotting hidden bugs :slightly_smiling_face: